examples/browser-only/package.json

```diff
@@ -15,6 +15,12 @@
     }
   },
   "dependencies": {
+    "@theia/ai-chat": "1.52.0",
```

Comment:
That's a good question. We think the packages (and their associated use cases) are succinct enough to warrant separation. For example code completion is by far the largest cost driver, for that reason alone it might not be included in all products.

Reply:
What do you mean by "cost driver"? Which cost? If an adopter does not want a feature, isn't it enough to simply unbind it via DI?

Reply:
> What do you mean by "cost driver"? Which cost?

The way it's implemented it uses a large amount of tokens. You basically send the whole current file to the LLM on every interaction.

> If an adopter does not want a feature, isn't it enough to simply unbind it via DI?

Yes, that could be done. However as a consequence the code will still end up in your application, bloating it for no reason and requiring filter logic. I thought of the current Theia runtime filtering mechanism is more of a crutch than the ideal we should strive for. I'm not sure where we should strike the balance. I guess that is a good topic for one of the next dev meetings.

Reply:
Good idea, I can put it on the agenda for next week. It would make sense for you to show up to the meeting, though.